package.json

```diff
@@ -6,8 +6,11 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "dotenv": "^10.0.0",
```

Comment:
do we need dotenv? create-react-app should be able to work with env without additional packages